examples/nlp_seq/train.py

```diff
@@ -109,6 +93,7 @@ def create_optimizer(model, learning_rate):
       weight_decay=FLAGS.weight_decay)
   optimizer = optimizer_def.create(model)
   optimizer = jax_utils.replicate(optimizer)
+  optimizer = optimizer.replicate()
```

Comment:
This is also deprecated, you should instead use jax_utils.replicate(optimizer)

Reply:
Done.